### PR TITLE
[12.0] Apply dotfiles

### PR DESCRIPTION
**Pull request by alexey-pelykh**

### CODE REVIEW

**Comment by alexey-pelykh** [comment]
@pedrobaeza I'm out of ideas why this one fails :|

**Comment by alexey-pelykh** [comment]
@sbidoul any ideas?

**Comment by sbidoul** [comment]
Not a first sight. A dependency issue maybe?

**Comment by alexey-pelykh** [comment]
The exact same code worked in v12 and works in v13 - I specifically tested, for some reason module listed in dependency seems not to be installed in new CI? But that's so unlikely

**Comment by sbidoul** [comment]
Ah, if you look at the install log, you see it got `odoo12-addon-project-role-12.0.1.1.1`, while it has (had) 12.0.3.0.0 in the repo.

I bumped version to 3.0.1. Can you try rebasing here?

In 15 and 16 there are no .dev versions anymore, and whenever merges are done with ocabot merge it's not an issue either since you get non-dev versions in these cases.

Question: why the `.po` files change in this PR?

**Comment by alexey-pelykh** [comment]
.po changes are mostly due to ```fuzzy, python-format``` "guessing" the wrong fuzzy substitution that has incorrect number of format elements in translation. I would guess fuzzy and formatting should be mutually exclusive but it appears no. Translations in some places are a mess

**Comment by sbidoul** [comment]
Hm, we shouldn't have fuzzy translations... I thought it was disabled in weblate.

Did you fix those manually or?

**Comment by alexey-pelykh** [comment]
Manually, editing translations to a meaningful-ish looking thing whenever the precommit complained

File: hr_timesheet_report/i18n/fr.po [hunk 1]
```diff
@@ -76,7 +76,6 @@ msgstr "Au moins un champ Détails doit être défini !"
 
 #. module: hr_timesheet_report
 #: code:addons/hr_timesheet_report/wizards/hr_timesheet_report_wizard.py:129
-#, fuzzy, python-format
```
Comment (alexey-pelykh) [inline hr_timesheet_report/i18n/fr.po:79]: @sbidoul or here, either msgid is outdated or it's not even a format string 🤷‍♂️

File: hr_timesheet_sheet/i18n/bg.po [hunk 2]
```diff
@@ -664,9 +664,8 @@ msgstr ""
 
 #. module: hr_timesheet_sheet
 #: code:addons/hr_timesheet_sheet/models/hr_timesheet_sheet.py:930
-#, fuzzy, python-format
 msgid "Save the Timesheet Sheet first."
-msgstr "График"
```
Comment (alexey-pelykh) [inline hr_timesheet_sheet/i18n/bg.po:669]: How that line was translated as "Schedule" is a mystery

**Comment by alexey-pelykh** [comment]
@sbidoul I will leave merging this PR I guess?

**Pull request merged**